### PR TITLE
Remove test_util/run-all script

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

This script was used by dcos-vagrant that was deprecated and repo was
removed. https://github.com/mesosphere-backup/dcos-vagrant

## Corresponding DC/OS tickets (required)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @drewvanstone @br-lewis @makkes @bamarni @rgo3 @orsenthil @branden

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required Remove deprecated and not used code

File: test_util/README.md [hunk 1]
```diff
@@ -2,5 +2,4 @@
 
 The dcos-integration-test harness and cloud deployment utilities that were previously hosted in this directory have been moved to: [DC/OS Test Utils](https://github.com/mesosphere/dcos-test-utils)
 
-Currently, this directory contains:
-* `run-all`: Checks out [DC/OS Vagrant](https://github.com/dcos/dcos-vagrant) and provides a DC/OS cluster from the onprem installer (`dcos_generate_config.sh`) sitting in the root of this repo (presumably generated by building DC/OS)
+Currently, this directory contains terraform helpers.
```
Comment (makkes) [inline test_util/README.md]: If I'm not blind this directory contains nothing when `run-all` is removed.

Comment (janisz) [reply test_util/README.md]: There are 4 other files https://github.com/dcos/dcos/tree/master/test_util
* README.md
* terraform_init.sh 	
* terraform_logs.sh
* terraform_test.sh

Comment (makkes) [reply test_util/README.md]: Ah, I was on the wrong branch locally. All good.

**Pull request merged**